priv/repo/migrations/20231220204054_add_override_for_test_groups.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Skate.Repo.Migrations.AddOverrideForTestGroups do
+  use Ecto.Migration
+
+  def change do
+    # excellent_migrations:safety-assured-for-this-file raw_sql_executed
```

Comment:
Just our luck - the first migration we write with `excellent_migrations` in place, and we basically have to ignore it 🤦